/home/mslu/.local/lib/python3.10/site-packages/pyftpdlib/authorizers.py:111: RuntimeWarning: write permissions assigned to anonymous user.
  self._check_permissions(username, perm)
[I 2024-10-25 21:54:54] concurrency model: async
[I 2024-10-25 21:54:54] masquerade (NAT) address: None
[I 2024-10-25 21:54:54] passive ports: None
[I 2024-10-25 21:54:54] >>> starting FTP server on 127.0.0.1:10021, pid=113343 <<<
[I 2024-10-25 21:54:55] 127.0.0.1:50434-[] FTP session opened (connect)
[I 2024-10-25 21:54:56] 127.0.0.1:50434-[anonymous] USER 'anonymous' logged in.
[I 2024-10-25 21:54:56] 127.0.0.1:50434-[anonymous] MKD /mnt/e/Courses_taking/Computer Networks/For Students/2019013240/client/src/client_test/test_vWUeYuWwnu 257
[I 2024-10-25 21:54:56] 127.0.0.1:50434-[anonymous] CWD /mnt/e/Courses_taking/Computer Networks/For Students/2019013240/client/src/client_test/test_vWUeYuWwnu 250
[I 2024-10-25 21:54:57] 127.0.0.1:50434-[anonymous] RETR /mnt/e/Courses_taking/Computer Networks/For Students/2019013240/client/src/client_test/test_vWUeYuWwnu/test_retr.data completed=1 bytes=80000 seconds=0.148
[I 2024-10-25 21:54:58] 127.0.0.1:50434-[anonymous] RETR /mnt/e/Courses_taking/Computer Networks/For Students/2019013240/client/src/client_test/test_vWUeYuWwnu/test_retr_2.data completed=1 bytes=80000 seconds=0.16
